core/arch/arm/plat-rzn1/platform_config.h

```diff
@@ -11,7 +11,7 @@
 
 /* DRAM */
 #define DRAM_BASE			0x80000000
-#define DRAM_SIZE			0x10000000
+#define DRAM_SIZE			0x40000000
```

Comment:
Maybe on a platform flavor basis?
Up to @b49020 recommendation.

Reply:
I would have better liked the DT based auto discovery of RAM approach, but since the proprietary boot-loader on RZN1 doesn't have the DT support yet and we have to live with it for the time being.

Also, having a different platform flavour just on the basis of different RAM size doesn't seems worth as compared to building platform specific binaries when a common tee.bin can work on both platforms.

AFAIK, the non-secure RAM is registered in OP-TEE just to service dynamic SHM requests from Linux. So just having a large enough memory space registered in OP-TEE such that it can handle differing RAM size requirements should be sufficient.

Reply:
@rfs613 Its worth to add a code comment for `DRAM_SIZE` to clarify that it supports both `256MB` and `1GB` RAM sizes.

Reply:
Done